marshmallow_recipe/options.py

```diff
@@ -17,6 +18,7 @@ class NoneValueHandling(enum.StrEnum):
 class DataclassOptions:
     none_value_handling: NoneValueHandling | None = None
```

Comment:
Maybe we should use this "MISSING" approach for all fields here then?
Kind of:
```python
class DataclassOptions:
    none_value_handling: NoneValueHandling = MISSING
    naming_case: NamingCase = MISSING
    decimal_places: int | None = MISSING
```

Reply:
Sure, we should!